src/plan/sticky/immix/global.rs

```diff
@@ -158,6 +159,14 @@ impl<VM: VMBinding> Plan for StickyImmix<VM> {
         self.gc_full_heap.load(Ordering::Relaxed) && self.immix.last_collection_was_exhaustive()
     }
 
+    fn current_gc_may_move_object(&self) -> bool {
+        if self.is_current_gc_nursery() {
+            !cfg!(feature = "sticky_immix_non_moving_nursery")
```

Comment:
This might be easier to understand and less error prone than directly checking the feature.
```suggestion
            crate::policy::immix_space::PREFER_COPY_ON_NURSERY_GC
```